lib/shelter.rb

```diff
@@ -1,2 +1,34 @@
+require 'pry'
 class Shelter
+    attr_reader :name, :capacity, :pets, :call_pets
+
+    def initialize(name, capacity)
+        @name = name
+        @capacity = capacity
+        @pets = []
+        @call_pets = []
```

Comment:
Because you have a fully defined method to return the pets with exclamation points, you don't need this instance variable (or the attr_reader for it above).